docs/docs/Best_Practices/.pages

```diff
@@ -7,3 +7,4 @@ arrange:
   - Unicode_Data.md
   - Working_with_Powershell.md
   - Scratch_Paths.md
+  - Metadata_and_Message_Limits.md
```

Comment:
Should we link to this page from anywhere?  For example, maybe from the schema section and/or the platform libraries section, we could have a quick blurb that says something like "Note that there are limits to how much data can be stored here, see (link) for details"?

Reply:
Good idea. Will add a link to this page in `User Visible Errors`, `Platform Libraries`, and `Logging`.